export function handleError(err){
    alert(`Something went wrong: ${err}`);
}